dali/ft/filecopy.cpp

```diff
@@ -1793,12 +1795,18 @@ void FileSprayer::gatherFileSizes(bool errorIfMissing)
 
 void FileSprayer::afterGatherFileSizes()
 {
-
     if (!copyCompressed)
     {
+        StringBuffer dateStr, timeStr, tailStr;
         ForEachItemIn(idx2, sources)
         {
             FilePartInfo & cur = sources.item(idx2);
+
+            LOG(MCdebugProgress, job, "%9d:%s (size: %llu bytes, last modified: %s %s)",
```

Comment:
picky: %9u more correct as idx2 is unsigned

Reply:
Changed